pkgs/tools/audio/beets/default.nix

```diff
@@ -46,13 +57,20 @@ lib.makeExtensible (self: {
   beets-minimal = self.beets.override { disableAllPlugins = true; };
 
   beets-unstable = callPackage ./common.nix {
-    inherit python3Packages extraPatches;
-    version = "2.2.0-unstable-2024-12-02";
+    inherit python3Packages;
+    extraPatches = [ ];
+    version = "2.2.0-unstable-2025-03-12";
     src = fetchFromGitHub {
       owner = "beetbox";
       repo = "beets";
-      rev = "f92c0ec8b14fbd59e58374fd123563123aef197b";
-      hash = "sha256-jhwXRgUUQJgQ/PLwvY1UfHCJ9UC8DcdBpE/janao0RM=";
+      rev = "670a3bcd17a46883c71cf07dd313fcd0dff4be9d";
+      hash = "sha256-hSY7FhpPL4poOY1/gqk7oLNgQ7KA/MJqx50xOLIP0QA=";
+    };
+    pluginOverrides = {
+      _typing = {
+        enable = true;
+        builtin = true;
+      };
```

Comment:
Hmm Maybe we should have a mechanism for enabling /disabling by default plugins depending on the version. I'm pretty sure I did something like that in the past in a certain state of stable v.s unstable. I _think_, that with this implementation, a `beets-unstable` user that is only interesting in adding / removing a few specific plugins, will also disable this plugin, but I'm not sure.

Reply:
I can add yet another plugin group for unstable-plugins. I didn't know if that was going too far.